src/components/TabBar/TabBar.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+
+interface Props {
+  children: React.ReactNode
+  bordered?: boolean
+  className?: string
+}
+
+type MergedProps = Props & InjectedProps
+
+const TabBarComponent: React.FC<MergedProps> = ({
+  className = '',
+  bordered = true,
+  children,
+  ...props
```

Comment:
[nits] The remaining props are only theme so there is no need to use the spread operator.